src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect, useMemo, useState } from 'react';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { preparePeopleArray } from '../../utils/preparePeopleArray';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[] | []>([]);
+  const [isLoaderActive, setIsLoaderActive] = useState(true);
+  const [isLoadingError, setIisLoadingError] = useState(false);
+
+  const fetchPeople = async () => {
+    try {
+      let data = await getPeople();
+
+      data = preparePeopleArray(data);
+      setPeople(data);
+      setIsLoaderActive(false);
+    } catch {
+      setIisLoadingError(true);
+    } finally {
+      setIsLoaderActive(false);
+    }
+  };
+
+  const peopleArrayIsAmpty = useMemo(() => {
+    return people.length === 0 && !isLoaderActive;
+  }, [people]);
```

Comment:
in this case, useMemo is not good, you don't perform complex operations with the object, such as filtering or mapping, comparing values is a simple operation, useMemo is overkill here